pages/docs/tutorials/kafka/managing-schemas-using-schema-registry.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Managing schemas using Schema Registry
+description: The guide explains how you can manage schemas using Schema Registry.
+weight: 50
+---
+
+## Introduction
+You will learn how to manage schemas with Schema Registry. Managing schemas will enable us to ensure a consistent data exchange across distributed systems, thus minimizing errors.
+
+## Background context
+The need for schema management has become increasingly prevalent to handle the evolving complexity of modern Event-Driven Architecture. A Schema Registry is a centralized service that stores and maintains the schemas for data exchanged between various components of a modern distributed system. Validating data being exchanged helps in maintaining data consistency and compatibility. 
+
+While several Schema Registry implementations exist, you will use the Apicurio Registry in this tutorial. Apicurio Registry is a popular open-source Schema Registry implementation that supports multiple serialization formats and facilitates schema management for diverse data in distributed systems. You will be using the Apicurio Registry in combination with Avro, which is a language-neutral data serialization system.
+
+## Installation guide
```

Comment:
I think you are referring to https://www.asyncapi.com/docs/tutorials/generate-code
`Installation guide` is there because there are actual installation instructions. You have prerequisites.

feel free to consult with other docs contributors and @alequetzalli - I of course can be wrong

Reply:
Hey @Arya-Gupta,

1. You are correct that our other tutorials **so far** have an `Installation Guide`, such as the one you and Lukasz are discussing (Generate code tutorial).
1. That said, Lukasz is also correct in pointing out that your tutorial is not detailing how to install Docker. You merely write a single sentence saying to install Docker, and you link to the Docker docs for instructions. Thus, Lukasz has a good point in suggesting you change this section to be a `Prerequisites`.

Please go ahead and make that tweak fix accordingly 😸

Reply:
Point noted! Thank you @alequetzalli @derberg